File: ERP/SKA/SKAOnline/SKA/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using SKA.Filters;

namespace SKA.Controllers
{
    public class HomeController : BaseController
    {
        [BranchCodeFilter]
        public ActionResult Index()
        {
            ViewBag.Message = "Welcome to ASP.NET MVC!";
            return View();
        }

        //[HttpPost]
        //public ActionResult Index(FormCollection form)
        //{

        //}

        public ActionResult About()
        {
            return View();
        }
    }
}

File: ERP/MDK_ERP/Controllers/HomeController.vb
Namespace MDK_ERP.Controllers

    <Authorize()> _
    Public Class HomeController
        Inherits System.Web.Mvc.Controller

        Function Index() As ActionResult
            Return RedirectToRoute("Production_default", New With {.controller = "Dashboard", .action = "Index"})
            Return View()
        End Function
    End Class

End Namespace

